sdk/webpubsub/Azure.Messaging.WebPubSub/api/Azure.Messaging.WebPubSub.netstandard2.0.cs

```diff
@@ -61,6 +61,7 @@ public partial class WebPubSubServiceClient
     public partial class WebPubSubServiceClientOptions : Azure.Core.ClientOptions
     {
         public WebPubSubServiceClientOptions(Azure.Messaging.WebPubSub.WebPubSubServiceClientOptions.ServiceVersion version = Azure.Messaging.WebPubSub.WebPubSubServiceClientOptions.ServiceVersion.V2021_05_01_preview) { }
+        public System.Uri ApiManagementEndpoint { get { throw null; } set { } }
```

Comment:
can this be a more generic naming such as ReverseProxyEndpoint? This can be ApiManagement and can also be other services supporting such request forwarding, for example, Amazon API Gateway...

Reply:
I changed it to ReverseProxyEndpoint